desktop/core/src/desktop/management/commands/rungunicornserver.py

```diff
@@ -93,6 +107,38 @@ def enable_logging(args, options):
   desktop.log.basic_logging(os.environ["HUE_PROCESS_NAME"])
   logging.info("Welcome to Hue from Gunicorn server " + HUE_DESKTOP_VERSION)
 
+
+if desktop.conf.TASK_SERVER_V2.ENABLED.get():
```

Comment:
Please avoid writing code directly to a python file. It is better to wrap the whole code in a function and then call it from wherever you need it. It makes for simpler maintainability, debugability and most importantly easier to unit-test.

Reply:
Wrapped this code into a function and then called it in start_server().